include/socgraph.php

```diff
@@ -11,6 +11,9 @@
 use Friendica\Core\Config;
 use Friendica\Network\Probe;
 
+use DomXPath;
+use DOMDocument;
```

Comment:
`PHP Warning:  The use statement with non-compound name 'DOMDocument' has no effect in include/socgraph.php on line 15`